CONTRIBUTING.md

```diff
@@ -15,19 +15,29 @@ This project uses `tox` for managing test environments. There are some pre-confi
 that can be used for linting and formatting code when you're preparing contributions to the charm:
 
 ```shell
-tox -e fmt           # update your code according to linting rules
 tox -e lint          # code style
 tox -e unit          # unit tests
 tox -e integration   # integration tests
 tox                  # runs 'lint' and 'unit' environments
 ```
```

Comment:
Have we missed static? Or am I missing something?

Reply:
✅